app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+from app.errors import NotVaccinatedError,\
+    OutdatedVaccineError, NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor):
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
Please, add descriptive messages for errors

Reply:
This function returns "Friends should buy 1 masks" instead of "All friends should be vaccinated"
for [not_wearing_mask, not_vaccinated] case

Reply:
```
(
    [
        {
            "name": "Alisa",
            "vaccine": {
                "name": "Pfizer",
                "expiration_date": datetime.date.today(),
            },
            "wearing_a_mask": False,
        },
        {
            "name": "Bob",
            "wearing_a_mask": True,
        },
    ],
    Cafe("KFC"),
    "All friends should be vaccinated",
),
```
Consider this test for the  `test_go_to_the_cafe` function